entity-framework/core/providers/sql-server/spatial.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Microsoft SQL Server Database Provider - Spatial Data - EF Core
+description: Using spatial data with the Entity Framework Core Microsoft SQL Server database provider
+author: bricelam
+ms.author: bricelam
+ms.date: 09/29/2020
+uid: core/providers/sql-server/spatial
+---
+# Spatial Data in the SQL Server EF Core Provider
+
+This page includes additional information about using spatial data with the Microsoft SQL Server database provider. For general information about using spatial data in EF Core, see the main [Spatial Data](xref:core/modeling/spatial) documentation.
+
+## Geography or geometry
+
+By default, spatial properties are mapped to `geography` columns in SQL Server. To use `geometry`, [configure the column type](xref:core/modeling/entity-properties#column-data-types) in your model.
+
+## Geography polygon rings
```

Comment:
Should this be a lower-level heading?

Reply:
I don't think so, it's not really related to choosing between geometry or geography